aries-site/src/examples/templates/forms/ShippingExample.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+import {
+  Box,
+  Button,
+  CheckBox,
+  Form,
+  FormField,
+  Header,
+  Heading,
+  Main,
+  MaskedInput,
+  Select,
+  Text,
+  TextInput,
+} from 'grommet';
+
+import { FormContainer } from '.';
+import { emailMask, emailValidation, phoneMask, states } from './formHelpers';
+
+export const ShippingExample = () => {
+  const [formValues, setFormValues] = React.useState({});
+
+  // eslint-disable-next-line no-unused-vars
+  const onSubmit = ({ value, touched }) => {
+    // Your submission logic here
+  };
+
+  return (
+    <FormContainer width="medium">
+      <Box gap="medium">
+        <Header
+          direction="column"
+          align="start"
+          gap="xxsmall"
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Heading level={3} margin="none">
+            Shipping
+          </Heading>
+          <Text>for your HPE products</Text>
+        </Header>
+        <Main
+          // Padding used to prevent focus from being cutoff
+          pad={{ horizontal: 'xxsmall' }}
+        >
+          <Form
```

Comment:
Noticing the formfields in the design have a black border as opposed to grey (I'm noticing a mix of grey and black formfield borders across the sticker sheet). For separate, we should reconcile which is the intended formfield border color.

Reply:
I hadn't caught this one yet! Added `- FormField
  - Border color (light #CCCCCC vs. #EEEEEE)` to https://github.com/hpe-design/design-system/issues/458